docs/features/modular_commands.rst

```diff
@@ -100,10 +100,10 @@ CommandSets and pass in the constructor to Cmd2.
             self._arg2 = arg2
 
         def do_show_arg1(self, _: cmd2.Statement):
-            self._cmd.poutput('Arg1: ' + self._arg1)
+            self._cmd.poutput('Arg1: %s' % self._arg1)
```

Comment:
Only a minor nit. Now that the minimum python version is 3.6 we've been favoring f-strings.